content/api/cypress-api/currentretry.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Cypress.currentRetry
+---
+
+`Cypress.currentRetry` is an number representing the current test retry count.
```

Comment:
Should there be a link to the [Test Retry docs](https://docs.cypress.io/guides/guides/test-retries)?

Reply:
Good call. I'll update the FAQ as well.